templates/data/spark/index.html

```diff
@@ -0,0 +1,129 @@
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h2 class="u-sv3">
+        Canonical’s complete, integrated and fully supported open source solution for data engineering and machine learning at scale.
+      </h2>
+      <ul>
+        <li>A complete, integrated, full-stack solution for Apache Spark®</li>
+        <li>Accelerate the time to value of your innovation projects</li>
+        <li>Founded on Kubernetes and Ubuntu</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2 class="u-sv3">
+      Accelerate your innovation projects at any scale
+    </h2>
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <h3>No friction</h3>
+      <p>Get started fast &mdash; the open source solution is free to use in any environment, in the data centre and in the cloud.</p>
+      <p><a href="https://snapcraft.io/spark-client" class="p-button">Try our Spark solution now</a></p>
+    </div>
+    <div class="col-4">
+      <h3>Expressway operations</h3>
+      <p>Rapidly deploy your Apache Spark® applications with the integrated, full stack solution from Canonical.</p>
+      <p><a href="https://juju.is/" class="p-button">Learn more about our solutions</a></p>
+    </div>
+    <div class="col-4">
+      <h3>Fully supported</h3>
+      <p>24/7 support and professional services are available to ensure your initiatives are always on track.</p>
+      <p><a href="/data/spark#get-in-touch" class="p-button">Contact sales</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>The best choice for Apache Spark®, anywhere</h2>
+      <p>Canonical’s scalable and comprehensive full stack system for Apache Spark covers:</p>
+      <ul>
+        <li>Security fixes and support for upstream Apache Spark.</li>
+        <li>Integrated, 24/7 supported solution founded on enterprise-proven Ubuntu Server LTS and Charmed Kubernetes.</li>
+        <li>Expert consulting services available when you need them.</li>
+      </ul>
+      <p><a href="/data/spark#get-in-touch" class="p-button--positive js-invoke-modal">Contact Sales</a></p>
+    </div>
+    <div class="col-5 u-hide--medium u-hide--small u-align--center u-vertically-center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/e8568793-contact+us.svg",
+        alt="",
+        width="144",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-7">
+      <h2>Expert services</h2>
+      <p>A production-grade, fully supported Spark environment deployed and operational, no fuss.</p>
+      <ul>
+        <li>Integrated solution on Charmed Kubernetes, Azure, and AWS.</li>
```

Comment:
Please adjust the indentation here so it aligns with the rest of the list elements